services/QuillLMS/config/application.rb

```diff
@@ -31,6 +31,7 @@ class Application < Rails::Application
     config.autoload_paths += %W(
       #{config.root}/app/controllers/concerns
       #{config.root}/lib
+      #{config.root}/app/lib
```

Comment:
Wasn't this already deployed? (It's not in `develop` although the actual directory `app/lib` is.). Not a big deal, since it does seem to be working in prod, just flagging in case it indicates a large commit/merge issue.

Reply:
Good question.  We moved various `lib/*.rb` files to app/lib in this commit `https://github.com/empirical-org/Empirical-Core/pull/7678` but I must have left this configuration out which would suggest this configuration is unnecessary.  I will remove it.